Automattic/wp-calypso
# Checkout: Move CheckoutTerms from old checkout to new checkout

#### Changes proposed in this Pull Request

This moves the `CheckoutTerms` component and all its dependencies from the old checkout directory (which is unused and will soon be removed) to the new checkout directory since it is used by new checkout.

In the process, it also cleans up some of the imports from the composite-checkout package. There were some places importing code from the package that wasn't part of the public API and other places that were using undocumented components.

<img width="569" alt="Screen Shot 2020-11-04 at 1 10 52 PM" src="https://user-images.githubusercontent.com/2036909/98152182-4b7c4200-1e9f-11eb-9465-edea1180e569.png">

#### Testing instructions

- Add a plan and a domain to your cart and visit checkout.
- Be sure you see the "By checking out, you agree..." section above the pay button in the last step.
